packages/optimistic-oracle/src/proposer.js

```diff
@@ -89,6 +89,14 @@ class OptimisticOracleProposer {
           return Array.isArray(x) && x.every((value) => typeof value === "string");
         },
       },
+      settleAllRequests: {
+        // `otherAccountsToSettle`: array of other accounts whose requests should be settled automatically.
+        value: false,
```

Comment:
Thanks for fixing this. looks much better now.

Reply:
Yeah, this is why I should actually run the tests before pushing :).